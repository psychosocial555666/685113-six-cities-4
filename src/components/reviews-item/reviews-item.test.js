import React from "react";
import renderer from "react-test-renderer";
import ReviewsItem from "./reviews-item.jsx";

const review = {
  reviewId: 14,
  reviewName: `Ozzy`,
  reviewAvatar: `src`,
  reviewGrade: 4,
  reviewText: `Id minim labore ut velit sit velit.Magna deserunt reprehenderit consequat elit cupidatat proident nostrud amet minim nulla.`,
  reviewTime: `May 8, 2016`,
};


it(`Should OfferCard render correctly`, () => {
  const tree = renderer
      .create(<ReviewsItem
        key={review.id * Math.random() + 1}
        review={review}
      />,
      {
        createNodeMock: () => {
          return {};
        }
      })
      .toJSON();

  expect(tree).toMatchSnapshot();
});
