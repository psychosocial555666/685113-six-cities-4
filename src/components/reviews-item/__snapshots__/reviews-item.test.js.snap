// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should OfferCard render correctly 1`] = `
<li
  className="reviews__item"
>
  <div
    className="reviews__user user"
  >
    <div
      className="reviews__avatar-wrapper user__avatar-wrapper"
    >
      <img
        alt="Reviews avatar"
        className="reviews__avatar user__avatar"
        height="54"
        src="src"
        width="54"
      />
    </div>
    <span
      className="reviews__user-name"
    >
      Ozzy
    </span>
  </div>
  <div
    className="reviews__info"
  >
    <div
      className="reviews__rating rating"
    >
      <div
        className="reviews__stars rating__stars"
      >
        <span
          style={
            Object {
              "width": "80%",
            }
          }
        />
        <span
          className="visually-hidden"
        >
          Rating
        </span>
      </div>
    </div>
    <p
      className="reviews__text"
    >
      Id minim labore ut velit sit velit.Magna deserunt reprehenderit consequat elit cupidatat proident nostrud amet minim nulla.
    </p>
    <time
      className="reviews__time"
      dateTime="2019-04-24"
    >
      May 8, 2016
    </time>
  </div>
</li>
`;
